src/services/socketService.ts

```
@@ -9,6 +9,10 @@ class SocketService {
   private maxReconnectAttempts = 5;
   private reconnectDelay = 1000;
   private isInitialized = false;
+  private hasLoggedConnectionError = false;
+  private isWebSocketDisabled = false;
+  private hasLoggedConnectionError = false;
```

Biome (2.1.2)

[error] Duplicate class member name "hasLoggedConnectionError"



(lint/suspicious/noDuplicateClassMembers)